### DISCUSSION TITLE
E-Mail with customized parameters

### DISCUSSION THREAD

**Discussion by lmarschall** [pocketbase/pocketbase#675] [Q&A] [answered]
Hey guys,

at the moment, I'm trying to use the default verification e-mail template and customize the HTML and parameters.
I was reading this discussion as well https://github.com/pocketbase/pocketbase/discussions/269, so I wondered if it is possible to provide your own variables like `{ORDER_NAME}` and send them via `client.users.requestVerification('mail@example.com', json_data)` in your json_data as parameters, but that did not work.

**Comment by ganigeorgiev** [comment, marked as answer]
No, this is not possible. Support for something similar may be added in the near future with the collection triggers (see #317).

But for now if you want to send an email with custom variables you'll have to extend PocketBase and use the `OnMailerBeforeUserVerificationSend` event hook - https://pocketbase.io/docs/event-hooks/#mailer-hooks

**Reply by lmarschall** [reply]
Hey @ganigeorgiev thank you, for your instant answer! Thought I could use the emails in a hacky way :) Hopefully support will come soon.
